### PR TITLE
[SW-1392] Document an example of training an AutoML model

### PR DESCRIPTION
**Pull request by mn-mikke**

### CODE REVIEW

File: py/pysparkling/ml/params.py [hunk 1]
```diff
@@ -623,7 +623,7 @@ def setStoppingMetric(self, value):
         return self._set(stoppingMetric=jvm.hex.ScoreKeeper.StoppingMetric.valueOf(correct_case_value))
 
     def setSortMetric(self, value):
-        assert_is_type(value, None, "AUTO", "deviance", "logloss", "MSE", "RMSE", "MAE", "RMSLE", "AUC" "mean_per_class_error")
+        assert_is_type(value, None, "AUTO", "deviance", "logloss", "MSE", "RMSE", "MAE", "RMSLE", "AUC", "mean_per_class_error")
```
Comment (jakubhava) [inline py/pysparkling/ml/params.py]: Nice catch!

File: doc/src/site/sphinx/tutorials/sw_automl.rst [hunk 2]
```diff
@@ -0,0 +1,163 @@
...
+            automl.setMaxModels(10)
+
+        Train the AutoML model. The training phase returns the best model according to the sortMetric. If it's not specified, the sortMetric is chosen automatically.
+
+        .. code:: scala
+
+            automl.setSortMetric("AUC")
+            val model = automl.fit(trainingDF)
+
+        You can also get raw model details by calling the *getModelDetails()* method available on the estimator as:
```
Comment (jakubhava) [inline doc/src/site/sphinx/tutorials/sw_automl.rst]: It is available on the model, no longer on the estimator

File: doc/src/site/sphinx/tutorials/sw_automl.rst [hunk 3]
```diff
@@ -0,0 +1,163 @@
...
+            automl.setMaxModels(10)
+
+        Train the AutoML model. The training phase returns the best model according to the sortMetric. If it's not specified, the sortMetric is chosen automatically.
+
+        .. code:: python
+
+            automl.setSortMetric("AUC")
+            model = automl.fit(trainingDF)
+
+        You can also get raw model details by calling the *getModelDetails()* method available on the estimator as:
```
Comment (jakubhava) [inline doc/src/site/sphinx/tutorials/sw_automl.rst]: same as above

File: doc/src/site/sphinx/tutorials/sw_automl.rst [hunk 4]
```diff
@@ -0,0 +1,163 @@
...
+        .. code:: python
+
+            model.transform(testingDF).show()
+
+
+
+Enabling XGBoost Models when Running Sparkling Water on YARN
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+The multi-node XGBoost algorithm is considered as an experimental feature of AutoML. Thus the XGBoost algorithm is disabled for AutoML by default when running
+Sparkling Water on YARN or a local cluster.
```
Comment (jakubhava) [inline doc/src/site/sphinx/tutorials/sw_automl.rst]: XGboost is disabled just when running sparkling water in distributed env. XGboost is enabled  in `local` mode

File: doc/src/site/sphinx/tutorials/sw_automl.rst (hunk 2 above)
Comment (mn-mikke) [reply doc/src/site/sphinx/tutorials/sw_automl.rst]: Good catch, thanks!

File: doc/src/site/sphinx/tutorials/sw_automl.rst (hunk 3 above)
Comment (mn-mikke) [reply doc/src/site/sphinx/tutorials/sw_automl.rst]: Good catch, thanks!

File: doc/src/site/sphinx/tutorials/sw_automl.rst [hunk 5]
```diff
@@ -128,21 +128,21 @@ The following sections describe how to train an AutoML model in Sparkling Water
 
         .. code:: python
 
-            model.transform(testingDF).show()
+            model.transform(testingDF).show(truncate = False)
 
 
 
 Enabling XGBoost Models when Running Sparkling Water on YARN
```
Comment (jakubhava) [inline doc/src/site/sphinx/tutorials/sw_automl.rst]: Maybe one note: People can run Sparkling Water in different kinds of distributed environments -> Kubernetes, Standalone Spark mode -> the same applies there

**Comment by mn-mikke** [comment]
@jakubhava I've tried to address your comments, can you have a second look?

**Pull request merged**